qa/tasks/util/__init__.py

```diff
@@ -258,3 +259,14 @@ def sudo_append_to_file(remote, path, data):
         ],
         stdin=data,
     )
+
+def get_rpm_pkg_version(remote, pkg):
+    """Gather RPM package version"""
+    version = None
+    cmd = 'rpm --queryformat="%{VERSION}" -q {}'.format(pkg)
+    try:
+        r = remote.sh(cmd, stdout=StringIO())
```

Comment:
Please read docs how to use remote.sh ;-) no need to use StringIO it does it for you.